src_c/surflock.c

```diff
@@ -99,7 +99,11 @@ pgSurface_LockBy(pgSurfaceObject *surfobj, PyObject *lockobj)
     if (surf->subsurface != NULL) {
         pgSurface_Prep(surfobj);
     }
+#if SDL_VERSION_ATLEAST(3, 0, 0)
+    if (!SDL_LockSurface(surf->surf)) {
+#else
     if (SDL_LockSurface(surf->surf) == -1) {
+#endif
```

Comment:
Not really an issue, but many simpler tools don't like unmatched parenthesis. A very simple solution to this problem is to move it out of the conditional preprocess:
```suggestion
#if SDL_VERSION_ATLEAST(3, 0, 0)
    if (!SDL_LockSurface(surf->surf))
#else
    if (SDL_LockSurface(surf->surf) == -1)
#endif
    {
```

Reply:
Good idea, I have applied your suggestion as a commit